.github/workflows/pull_request_previews.yml

```diff
@@ -0,0 +1,27 @@
+name: Synchronize Pull Request Previews
+on:
+  schedule:
+    - cron: 0/5 * * * *
+jobs:
+  update-pr-preview:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install dependency
+      run: pip install requests
+    - name: Synchronize state
+      run:
+        ./tools/ci/pr_preview.py
+          --host https://api.github.com
+          --github-project web-platform-tests/wpt
+          synchronize
+          --window 480
+      env:
+        # This Workflow must trigger further workflows. The GitHub-provided
+        # `GITHUB_TOKEN` secret is incapable of doing this [1], so a
+        # user-generated token must be specified instead. This token requires
+        # the "repo" scope, and is should be stored as a Secret named
```

Comment:
```suggestion
        # the "repo" scope, and is should be stored as a secret named
```

Reply:
The documentation in this patch consistently references proprietary GitHub concepts with capital letters. I prefer this because it leaves us room to talk about things like "preview deployments" as distinct from the resources provided by GitHub. If you and the other maintainers aren't swayed by that, then I will remove all relevant capitalization.

Reply:
The documentation in this patch consistently references proprietary GitHub concepts with capital letters. I prefer this because it leaves us room to talk about things like "preview deployments" as distinct from the resources provided by GitHub. If you and the other maintainers aren't swayed by that, then I will remove all relevant capitalization.

Reply:
The documentation in this patch consistently references proprietary GitHub concepts with capital letters. I prefer this because it leaves us room to talk about things like "preview deployments" as distinct from the resources provided by GitHub. If you and the other maintainers aren't swayed by that, then I will remove all relevant capitalization.

Reply:
I've you've done this consistently then OK, I won't hunt down all the capital letter. I think it looks like a typo and suspect I'm not alone, but let's wait for a second person to try correcting it before doing anything.

Reply:
Apologies, I didn't mean to suggest that you should find and replace the capitalization. If my rationale doesn't sway folks, then I'll certainly update the patch.